addons/sale/views/account_views.xml

```diff
@@ -44,6 +44,11 @@
                     <field name="source_id"/>
                 </group>
             </xpath>
+            <xpath expr="//div[@name='button_box']" position="inside">
+                <button class="oe_stat_button" name="action_view_source_sale_orders" type="object" icon="fa-pencil-square-o" attrs="{'invisible': ['|', ('sale_order_count', '=', 0), ('move_type', '!=', 'out_invoice')]}">
```

Comment:
same as purchase, why not showing this button when there are linked orders, no matter if it is an in_invoice or not?